src/metabase/driver/generic_sql.clj

```diff
@@ -454,7 +448,8 @@
    :excluded-schemas     (constantly nil)
    :field->identifier    (u/drop-first-arg (comp (partial apply hsql/qualify) field/qualified-name-components))
    :field->alias         (u/drop-first-arg name)
-   :field-percent-urls   fast-field-percent-urls
+;   :field-percent-urls   fast-field-percent-urls
+;   :field-avg-length     (u/drop-first-arg field-avg-length)
```

Comment:
why is this all getting commented out? Do we not need it anymore? (If so, why not just delete the lines altogether?)